features/sdk-node-properties.js

```diff
@@ -0,0 +1,247 @@
+const escape = require('escape-html');
+
+const isPropertyKey = (key) => key.startsWith('.');
+const propertyKeyName = (key) => key.substring(1);
+
+const specificationPointRegExp = /^[A-Z]{1,3}[1-9]\d*([a-z]((([1-9]\d*)?[a-z])?([1-9]\d*)?)?)?$/;
```

Comment:
I've not fully validated this against your test suite, but the regex `/^[A-Z]{1,3}[1-9]\d*([a-z]$|[a-z][0-9]|[0-9]*)*$/` should match the spec points the same as this, with the added benefit that 'RTE6aa/RTE6aa1' are no longer valid, and the nesting is not limited (if that is a good thing or not I'm not sure)

Reply:
The one thing I do know about regular expressions is that there will always be somebody who writes them a lot better than I do. Thanks! 😁

Reply:
I just tried the suggested new regular expression (`/^[A-Z]{1,3}[1-9]([a-z]?\d)*[a-z]?$/`) locally and it is, indeed, catching `RTE6aa` and `RTE6aa1` as invalid now. 🎉 Unfortunately it's also now failing to catch `A1b02`, `A1b2c03` and `A1b2c3d` as invalid. 😢 

Entering them into [the regex101 page you linked](https://regex101.com/r/qwD70f/2) does seem to work as expected for all of the above, so I'm now really confused. I may loop back to this later, but for now I'm going to set this aside so I can look at other changes needed.

Reply:
That's weird, I guess there must be some discrepancy in regex parsing here that is too obscure for mere mortals to understand.